### PR TITLE
Add basic steps for accessing UI and Cluster Nodes

### PR DESCRIPTION
**Pull request by sgryczan**
Update README with basic post-installation steps.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,3 +114,60 @@ quay, or gcr and push dev builds to your own registry if you like by running
...
+rancher_server_url = https://<Your Rancher URL>
+```
+
+## Accessing the Rancher UI
+
+In order to access the Rancher UI via your chosen `rancher_server_url`, you must ensure that a valid DNS record exists, that resolves to one or more of your cluster node IPs.
+
+Once the DNS record is in place, the Rancher UI will become accessible at the location specified for `rancher_server_url`.
+
+#### Automatic DNS Names
```
Comment (bgentry-ntap) [inline README.md:147]: Some of this is repeated above in `Supplemental DNS names` above. Do we want this both places? I'm good either way

https://github.com/NetApp/ez-rancher/pull/52/files#diff-04c6e90faac2675aa89e2176d2eec7d8R38

File: README.md [hunk 2]
```diff
@@ -115,3 +114,60 @@ quay, or gcr and push dev builds to your own registry if you like by running
...
+```bash
+$ export KUBECONFIG=`pwd`/deliverables/kubeconfig
+$ kubectl get nodes
+NAME             STATUS   ROLES              AGE     VERSION
+node01   Ready    controlplane,etcd,worker   5m30s   v1.17.5
+node02   Ready    controlplane,etcd,worker   5m30s   v1.17.5
+node03   Ready    controlplane,etcd,worker   5m30s   v1.17.5
+```
+
+### SSH
```
Comment (bgentry-ntap) [inline README.md:169]: This is repeated above in the Deliverables section. I think this should be consolidated.

File: README.md (hunk 1 above)
Comment (sgryczan) [reply README.md:168]: I updated `Supplemental DNS names` to link to the `Automatic DNS names` section

File: README.md (hunk 2 above)
Comment (sgryczan) [reply README.md:190]: I put a link in the deliverables section, that will take users to the `Admin Access to Cluster Nodes` section that contains more details

**Pull request merged**